mssqlcli/main.py

```diff
@@ -648,9 +651,14 @@ def get_completions(self, text, cursor_positition):
             return self.completer.get_completions(
                 Document(text=text, cursor_position=cursor_positition), None)
 
-    def get_prompt(self):
+    def get_prompt(self, string):
         # mssql-cli
-        string = self.mssqlcliclient_query_execution.database + u'>'
+        string = string.replace('\\t', self.now.strftime('%x %X'))
+        string = string.replace('\\u', self.mssqlcliclient_query_execution.user_name or '(none)')

```

Comment:
If the `-E` options is specified, we use windows/kerberos, which defaults to the current user. This is really common for Windows environments. Can we detect if -E was passed, and get the current user via `os.getlogin()`?

Reply:
If you don't have a Windows environment to test this, feel free to skip this, and someone with a Windows machine can fix this in another PR.